lib/codegen.js

```diff
@@ -114,6 +114,10 @@ var getViewForSwagger2 = function(opts, type){
                     });
                 }
 
+                if (parameter.required === false) {
+                    parameter.isOptional = true
+                }
```

Comment:
`parameter.isOptional = !parameter.required`? Can the existing code differentiate on the inverse of `.required`?

Reply:
I'm not sure if it's safe to assume that every method parameter is going to have a `required` property. I'd rather keep the behavior if it's `undefined` the same as before for now.